src/controllers/util/conversionStrategies/strategyV1ToV2.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  SourceInput,
+  SourceInputConversionResult,
+  SourceInputV2,
+  SourceRequestV2,
+} from '../../../types';
+import { VersionConversionStrategy } from './abstractions';
+
+export class StrategyV1ToV2 extends VersionConversionStrategy<SourceInput, SourceInputV2> {
+  convert(sourceEvents: SourceInput[]): SourceInputConversionResult<SourceInputV2>[] {
+    // Currently this is not being used
+    // Hold off on testing this until atleast one v2 source has been implemented
+    return sourceEvents.map((sourceEvent) => {
+      try {
+        const sourceRequest: SourceRequestV2 = {
+          method: '',
```

Comment:
do you need to send empty values
?

Reply:
@koladilip Keeping in alignment with null values in `go`, I went with these empty values. The motivation behind this is that, SourceRequestV2 will necessarily have empty values when they come from rudder-server and I am mimicking those default values in v1tov2 conversion. 
Let me know your thoughts here.